### ISSUE TITLE
All-day events from Google Calendar

### ISSUE THREAD

**Issue by radical-equanimity** [CraigT543/EasyBlue#21]
The sync feature doesn't seem to pull in all-day events from Gmail. I booked a multi-night hotel stay and Gmail automatically parsed my confirmation email to create an event on my calendar. If I'm on travel I'm not available so I'd like to be able to sync these automatically generated "all-day" events from Gmail.

**Comment by CraigT543** [comment]
Yes this is a missing piece of code.  What can work is to do the all day event in EA and it will sync that way.  We are missing the code going back so if you start it in Google it will not sync.  This needs to be scouted out and added.  What I have done is taken an appointment event in google and scheduled for the full work day (11:00 to 19:00 for example) and duplicated it for the number of days and that syncs on both sides as it is now.

**Comment by radical-equanimity** [comment]
I've noticed that my Google event starting at 8am local time is duplicated in my google calendar as an event titled "unavailable" but scheduled 4 hours earlier as if it were using GMT instead of local time zone.
